src/sourmash/sourmash_args.py

```diff
@@ -156,6 +159,46 @@ def report_picklist(args, picklist):
             sys.exit(-1)
 
 
+def load_include_pattern(args):
+    if args.picklist and (args.include_db_pattern or args.exclude_db_pattern):
+        error("ERROR: --picklist and --include/--exclude cannot be used together.")
+        sys.exit(-1)
+
```

Comment:
is there an underlying reason why using both together would be inadvisable (other than, we don't see an immediate use for it and it would likely complicate code a bit, so it's not implemented)? Just curious, not recommending adding! Same q below for `include/exclude`.

Reply:
an earlier version of the error message said "for now". doing it this way leaves it open for us to expand the option later, but I worry that the tests etc get hairy so I wanted to just do it this way for now.

the other statement is, at some point life gets complicated enough that you should just use picklists, I think? 😆